### DISCUSSION TITLE
Newer dinput8.dll version not loading TemporalUpscaler.

### DISCUSSION THREAD

**Discussion by snoopi85** [praydog/REFramework#915] [General Questions]
As stated, if I use newer versions of dinput8.dll, TemporalUpscaler will not show at all in the framework menu.

But if I use much older dinput8.dll version, TemporalUpscaler will show in the framework menu and will work.

Could anyone help me here, why does the newer dinput8.dll not show the TemporalUpscaler mod for me?

Thanks!

**Comment by praydog** [comment]
Use the builds that say `pd-upscaler` next to them. https://github.com/praydog/REFramework/actions

**Comment by snoopi85** [comment]
OK got it working. You are a legend sir, thank you so much!
